configuration/ldap/ldap_config.py

```diff
@@ -77,7 +77,8 @@ def _import_group_type(group_type_name):
 
 # For more granular permissions, we can map LDAP groups to Django groups.
 AUTH_LDAP_FIND_GROUP_PERMS = environ.get('AUTH_LDAP_FIND_GROUP_PERMS', 'True').lower() == 'true'
-AUTH_LDAP_MIRROR_GROUPS = environ.get('AUTH_LDAP_MIRROR_GROUPS', '').lower() == 'true'
+if environ.get('AUTH_LDAP_MIRROR_GROUPS') is not None:
```

Comment:
I think this is not necessary, as explained here: https://github.com/netbox-community/netbox-docker/issues/445#issuecomment-784373177

Reply:
Once https://github.com/netbox-community/netbox-docker/pull/450 is merged, I don't think this change will be necessary anymore. I would still like to keep the PR for the new `extra.py` file in the `ldap` folder.

Reply:
I believe this is not resolved either. Since #450 is now merged, the changes on these two lines should not be required any longer.